scripts/rook.sh

```diff
@@ -7,7 +7,7 @@ ROOK_BLOCK_POOL_NAME=${ROOK_BLOCK_POOL_NAME:-"newrbdpool"}
 ROOK_CEPH_CLUSTER_VERSION="v14.2.10"
 
 rook_version() {
-	echo "${ROOK_VERSION#?v}" | cut -d'.' -f"${1}"
+	echo "${ROOK_VERSION#?}" | cut -d'.' -f"${1}"
```

Comment:
why leave the `#$` in here?

Reply:
@nixpanic in case of first iteration on the string `v14.2.10` it was giving back `v1`.

Reply:
Ah, yes. But in this case you are cutting the 1st character from the ROOK_VERSION, which may not be the `v` for all (future) release tags. It is a little safer to use `echo "${ROOK_VERSION#v}"` that will only drop an initial `v`.

Reply:
Sure. done.. ptal